### PR TITLE
Add support column reference in LOAD statement

### PR DESCRIPTION
**Pull request by imay**

### CODE REVIEW

File: fe/src/main/java/org/apache/doris/analysis/DataDescription.java [hunk 1]
```diff
@@ -216,6 +244,12 @@ private void checkColumnMapping() throws AnalysisException {
 
             parsedExprMap.put(column, child1);
 
+            if (!(child1 instanceof FunctionCallExpr)) {
+                // only just for pass later check
+                columnToFunction.put(column, Pair.create("__slot_ref", Lists.newArrayList()));
```
Comment (EmmyMiao87) [inline fe/src/main/java/org/apache/doris/analysis/DataDescription.java]:
```suggestion
                columnToFunction.put(column, Pair.create("slot_ref", Lists.newArrayList()));
```

File: fe/src/main/java/org/apache/doris/load/BrokerFileGroup.java [hunk 2]
```diff
@@ -129,10 +131,13 @@ public void parse(Database db) throws DdlException {
...
             lineDelimiter = "\n";
         }
+
+        fileFormat = dataDescription.getFileFormat();
+
         isNegative = dataDescription.isNegative();
 
         // FilePath
-        filePathes = dataDescription.getFilePathes();
+        filePaths = dataDescription.getFilePathes();
```
Comment (EmmyMiao87) [inline fe/src/main/java/org/apache/doris/load/BrokerFileGroup.java]:
```suggestion
        filePaths = dataDescription.getFilePaths();
```

File: fe/src/main/cup/sql_parser.cup [hunk 3]
```diff
@@ -198,7 +198,7 @@ terminal String KW_ADD, KW_ADMIN, KW_AFTER, KW_AGGREGATE, KW_ALL, KW_ALTER, KW_A
     KW_DATA, KW_DATABASE, KW_DATABASES, KW_DATE, KW_DATETIME, KW_DECIMAL, KW_DECOMMISSION, KW_DEFAULT, KW_DESC, KW_DESCRIBE,
     KW_DELETE, KW_DISTINCT, KW_DISTINCTPC, KW_DISTINCTPCSA, KW_DISTRIBUTED, KW_DISTRIBUTION, KW_BUCKETS, KW_DIV, KW_DOUBLE, KW_DROP, KW_DROPP, KW_DUPLICATE,
     KW_ELSE, KW_END, KW_ENGINE, KW_ENGINES, KW_ENTER, KW_ERRORS, KW_EVENTS, KW_EXISTS, KW_EXPORT, KW_EXTERNAL, KW_EXTRACT,
-    KW_FALSE, KW_FOLLOWER, KW_FOLLOWING, KW_FREE, KW_FROM, KW_FIRST, KW_FLOAT, KW_FOR, KW_FRONTEND, KW_FRONTENDS, KW_FULL, KW_FUNCTION,
+    KW_FALSE, KW_FOLLOWER, KW_FOLLOWING, KW_FREE, KW_FROM, KW_FIRST, KW_FLOAT, KW_FOR, KW_FORMAT, KW_FRONTEND, KW_FRONTENDS, KW_FULL, KW_FUNCTION,
```
[the remark this answers is not in the document]
Comment (imay) [reply fe/src/main/cup/sql_parser.cup]: Yes, you are right

File: fe/src/main/java/org/apache/doris/load/BrokerFileGroup.java (hunk 2 above)
Comment (imay) [reply fe/src/main/java/org/apache/doris/load/BrokerFileGroup.java]: OK

**Review by morningman — APPROVED** [review]

**Pull request merged**